### PR TITLE
[Beta] Use hunspell dictionaries from the host filesystem.

### PR DESCRIPTION
**Pull request by nteodosio**
Imitates Firefox' setup as in [1], ignoring compatibility stuff pertaining to the host-hunspell interface, which Thunderbird never had.

[1] https://github.com/canonical/firefox-snap/pull/53/

-------

### Verification

I had these hunspell dictionaries installed:

    hunspell-de-ch/oracular,now 20161207-12 all  [installiert]
    hunspell-en-us/oracular,now 1:2020.12.07-2 all  [installiert]

If I install the snap hereby produced, connect the interface and re-install it, I get in the `snap run --shell thunderbird`:

```
$ ls -la $SNAP_COMMON/snap-hunspell
total 8
drwxr-xr-x 2 root root 4096 Sep 19 09:55 .
drwxr-xr-x 4 root root 4096 Sep 19 09:53 ..
lrwxrwxrwx 1 root root   49 Sep 19 09:55 de_CH.aff -> /snap/thunderbird/x2/usr/share/hunspell/de_CH.aff
lrwxrwxrwx 1 root root   49 Sep 19 09:55 de_CH.dic -> /snap/thunderbird/x2/usr/share/hunspell/de_CH.dic
lrwxrwxrwx 1 root root   49 Sep 19 09:55 de_LI.aff -> /snap/thunderbird/x2/usr/share/hunspell/de_LI.aff
lrwxrwxrwx 1 root root   49 Sep 19 09:55 de_LI.dic -> /snap/thunderbird/x2/usr/share/hunspell/de_LI.dic
lrwxrwxrwx 1 root root   49 Sep 19 09:55 en_US.aff -> /snap/thunderbird/x2/usr/share/hunspell/en_US.aff
lrwxrwxrwx 1 root root   49 Sep 19 09:55 en_US.dic -> /snap/thunderbird/x2/usr/share/hunspell/en_US.dic
```

If I now do

    apt remove hunspell-en-us

and re-install the snap, I get

```
$ ls -la $SNAP_COMMON/snap-hunspell
total 8
drwxr-xr-x 2 root root 4096 Sep 19 09:56 .
drwxr-xr-x 4 root root 4096 Sep 19 09:53 ..
lrwxrwxrwx 1 root root   49 Sep 19 09:56 de_CH.aff -> /snap/thunderbird/x3/usr/share/hunspell/de_CH.aff
lrwxrwxrwx 1 root root   49 Sep 19 09:56 de_CH.dic -> /snap/thunderbird/x3/usr/share/hunspell/de_CH.dic
lrwxrwxrwx 1 root root   49 Sep 19 09:56 de_LI.aff -> /snap/thunderbird/x3/usr/share/hunspell/de_LI.aff
lrwxrwxrwx 1 root root   49 Sep 19 09:56 de_LI.dic -> /snap/thunderbird/x3/usr/share/hunspell/de_LI.dic
```

### CODE REVIEW

File: snap/hooks/post-refresh
Comment (seb128) [inline snap/hooks/post-refresh]: Firefox->Thunderbird

**Review by seb128 — CHANGES_REQUESTED** [review]
Thanks, that seems fine to be an is basically what we are already doing in Firefox. I've one nitpick about Firefox->Thunderbird in the script comments

File: snap/hooks/post-refresh
Comment (nteodosio) [reply snap/hooks/post-refresh]: Thanks, fixed.

**Review by seb128 — APPROVED** [review]
Thanks. Seems fine to me but I will wait on Amin's review before merging

**Review by bandali — APPROVED** [review]
LGTM as well, thanks.

**Pull request merged**
